docs/basics/mutating-data.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Mutations
+order: 2
+---
+
+## Mutate your data
+
+Now that we know how to query our data we'll also need to know
+how to mutate it, for this the `useMutation` hook and the `Mutation`
+component are exported.
+
+Let's set up a mutation allowing us to change the name of our todo.
+
+```jsx
+const Todo = ({ id, title }) => {
+  const [updateTodoResult, updateTodo] = useMutation(`
+    mutation ($id: ID!, $title: String!) {
+      updateTodo (id: $id, title: $title) {
+        id
+      }
+    }
+  `);
+}
+```
+
+Similar to the `useQuery` output, `useMutation` returns a tuple. The first item in the tuple being our `result`
+containing: `fetching`, `error`, and `data`. At this point in time, no mutation has been performed.
+To mutate the data we first have to invoke the second item in the tuple - the function here named `updateTodo`.
+
+### Using the result
+
+When calling this function we have two ways of getting the response from the server,
+we can get it from `updateTodoResult` or we chain a `.then` on our `updateTodo` call.
```

Comment:
Minor wording change!


```suggestion
we can get it from `updateTodoResult` or we can await the promise returned from our mutation trigger function.
```

Also, is it fair to assume most will be using async/await?

Reply:
Agreed, let’s not make any assumptions and in general just state that it returns a result in the expected, identical shape 👍

It’s a minor detail but whenever possible it’s good to state all options and details of an API of ours whenever possible (we can add cross-references / links to API / other pages later) but stop whenever an explanation dives outside of urql / GraphQL